### DISCUSSION TITLE
The Suez Canal is approximately how long?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#656] [Galaxy Trivia] [answered]
**Bonus:** What "waters" does it connect?

**Comment by DevElkami** [comment]
193.30 km (120.11 mi) long.

Connect the Mediterranean Sea to the Red Sea

**Comment by peterbaccaro** [comment]
The Suez Canal is approximately 120 miles (193 kilometers) long.
The Suez Canal connects the Mediterranean Sea to the Red Sea, providing a direct shipping route between Europe and Asia without the need to navigate around the southern tip of Africa.

**Comment by FleetAdmiralJakob** [comment, marked as answer]
The Suez Canal is approximately 120 miles (193 kilometers) long. It connects the Mediterranean Sea to the Red Sea, allowing ships to travel between Europe and Asia without having to go around the southern tip of Africa. The canal is located in Egypt and was completed in 1869.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/FleetAdmiralJakob?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Comment by magnandicko93-crypto** [comment]
Ohdhhd

**Comment by magnandicko93-crypto** [comment]
Hzbz

**Comment by magnandicko93-crypto** [comment]
Bznq

**Comment by magnandicko93-crypto** [comment]
magnandicko93@gmail.com
